src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts

```diff
@@ -784,6 +786,18 @@ export class ReplicationFormComponent {
                 }
             }
         )
+
+        entityForm.formGroup.controls['speed_limit'].valueChanges.subscribe((value) => {
+            const speedLimitField = _.find(this.fieldConfig, {name: "speed_limit"});
+            const filteredValue = this.storageService.convertDataMeasurementStrings(value);
+                    if (isNaN(filteredValue)) {
+                        speedLimitField['hasErrors'] = true;
```

Comment:
These two fields could be combined, using the presence of a string as the boolean value. In other words:
```
if (speedLimitField['errors']) { print('there are errors') }
```

Reply:
If I understand, 'errors' and 'hasErrors' are to separate properties that have to be set on our form field.